### DISCUSSION TITLE
Credit of trust feature for players and trusted player rank

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#729] [Ideas]
I suggest allowing every member who reached a certain cap of points to honor any member they respect with credit of trust. The number of granted credits would be displayed on the user profile, and on global leaderboard. The number should contain a link to the section which will diplay all users, who brought credit. I think this feature should only apply to the quality of achievement earning performance, and be named "# of members honored this player with credit of trust", and on global leaderboard could be named "trust for player" or something similiar with an emphasis on player. Players with big number of honets credits, and verified good opinion in the community, could be promoted to the rank of "trusted player".

**Comment by SyrianBallaS** [comment]
I think it's a good idea. Too many people like to cheat these days including professional gamers (as-in playing for money).

**Comment by kdecks** [comment]
It's a fun idea, but I don't see it as an effective method for curbing cheating or identifying "good" players. In practice I see it as a popularity feature, as there's nothing that would stop a "trusted" member from cheating or to have already cheated in the past.

Even if you've see clean streams from a reliable user, that doesn't mean they've never cheated. I don't object to something like this existing in a slightly different form such as liking a player or promoting other players as a sort of tag.

**Comment by julkasalsa** [comment]
Everything can be use as a popularity feature if you just show numbers. I would insist on making each credit visible, to display who actually trust specific person.
It's not made for users to show off (I won't deny it could me used for that), but to give them a chance to show who they trust. Eventually we could remove the number, just add a link to the list of users.

**Comment by kdecks** [comment]
My main point is that it's not a good indication of cheating or the lack of it.

**Comment by julkasalsa** [comment]
May main point is you see it as popularity feature, but wouldn't object "liking a player" which is absolute form of popularity feature. And I don't said anything about cheat indication there. It's based on everyones own will to show who they trust.

**Comment by kdecks** [comment]
And if a player is more trusted what does that indicate? 

Also you're right, I can't imagine something like this being used in any other way that "liking" a user.

**Comment by julkasalsa** [comment]
I guess you see everything like facebook. It's not about number of likes, but who is trusting in detail. For me for example most of those trust credits wouldn't mean much. Yeah a lot of users would see it as kind of popularity meter, but for some only certain credits will matter.
